interfaces/builtin/screen_inhibit_control.go

```diff
@@ -26,6 +26,7 @@ import (
 const screenInhibitControlConnectedPlugAppArmor = `
 # Description: Can inhibit and uninhibit screen savers in desktop sessions.
 #include <abstractions/dbus-session-strict>
+#include <abstractions/dbus-strict>
```

Comment:
Ah, you are right. Odd that is is on the system bus, but by all means, since the unity screen API in the policy very clearly is on the system bus, we should add `#include <abstractions/dbus-strict>`. Thanks!

+1